src/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs

```diff
@@ -8,49 +8,27 @@
 
 namespace System.Net.Sockets
 {
-    /// <devdoc>
-    ///    <para>
-    ///       Provides the underlying stream of data for network access.
-    ///    </para>
-    /// </devdoc>
+    // Provides the underlying stream of data for network access.
     public class NetworkStream : Stream
     {
-        /// <devdoc>
-        ///    <para>
-        ///       Used by the class to hold the underlying socket the stream uses.
-        ///    </para>
-        /// </devdoc>
+        // Used by the class to hold the underlying socket the stream uses.
         private Socket _streamSocket;
 
-        /// <devdoc>
-        ///    <para>
-        ///       Used by the class to indicate that the stream is m_Readable.
-        ///    </para>
-        /// </devdoc>
+        // Used by the class to indicate that the stream is m_Readable.
         private bool _readable;
 
-        /// <devdoc>
-        ///    <para>
-        ///       Used by the class to indicate that the stream is writable.
-        ///    </para>
-        /// </devdoc>
+        // Used by the class to indicate that the stream is writable.
         private bool _writeable;
 
         private bool _ownsSocket;
 
-        /// <devdoc>
-        /// <para>Creates a new instance of the <see cref='System.Net.Sockets.NetworkStream'/> without initalization.</para>
-        /// </devdoc>
+        // Creates a new instance of the System.Net.Sockets.NetworkStream without initalization.
```

Comment:
I'm putting this off until we decide on a documentation strategy for System.Net. The edits done here are consistent with what was done in other S.N assemblies.